datamade/la-metro-councilmatic
# Add banners with remote meeting instructions

## Overview

See title.

### Checklist


### Demo

<img width="1440" alt="Screen Shot 2020-04-14 at 10 04 31 AM" src="https://user-images.githubusercontent.com/12176173/79240431-6326a700-7e37-11ea-8882-12c59ea706fe.png">

## Testing Instructions

 * View the homepage and confirm the banners look nice.
 * Click on the English and Spanish links and confirm they resolve to the correct location.

Handles #601
